### PR TITLE
e2e false positives

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

Unfortunately, not all the green we see in the e2e testing is actually green.

This PR makes the output of the e2e testing more reliable.

### CODE REVIEW

File: tests/e2e/tutorials/tutorialBase.js [hunk 1]
```diff
@@ -23,7 +23,13 @@ class TutorialBase {
   }
 
   initScreenshoter() {
-    utils.createScreenshotsDir();
+    try {
+      utils.createScreenshotsDir();
```
Comment (pcrespov) [inline tests/e2e/tutorials/tutorialBase.js:27]: why would it fail creating an empty directory  in CI?

File: tests/e2e/tutorials/tutorialBase.js [hunk 2]
```diff
@@ -35,12 +41,18 @@ class TutorialBase {
...
-    if (!needsRegister) {
-      await this.login();
+      const needsRegister = await this.registerIfNeeded();
+      if (!needsRegister) {
+        await this.login();
+      }
+    }
+    catch(err) {
+      console.error("Error starting", err);
+      throw(err);
```
Comment (pcrespov) [inline tests/e2e/tutorials/tutorialBase.js:55]: If you throw an exception, cannot you write the stack when caught??

**Review by sanderegg — APPROVED** [review]
ay but we're going to be red with this :rofl:

File: tests/e2e/tutorials/tutorialBase.js (hunk 2 above)
Comment (odeimaiz) [reply tests/e2e/tutorials/tutorialBase.js:55]: So far I didn't find it necessary.

File: tests/e2e/tutorials/tutorialBase.js (hunk 1 above)
Comment (odeimaiz) [reply tests/e2e/tutorials/tutorialBase.js:27]: Permissions, wrong path... It's just a matter of keeping everything under try/catches. It doesn't hurt, does it?

**Review by GitHK — APPROVED** [review]

**Pull request merged**